C_CPP/Data Structures/Stack/implementation_of_stack_using_queue.md

```diff
@@ -0,0 +1,97 @@
+//Our aim is to implement stack using queue
+
+#include <bits/stdc++.h>
```

Comment:
Take your time & format this correctly...
Why's the code outside codeblocks & documentation in comments...

Also, this needs proper explanation & documentation not just the code